src/v2/guide/single-file-components.md

```diff
@@ -52,20 +52,22 @@ Incluso si no le gusta la idea de utilizar Single-File Components, usted puede a
 
 ### Ejemplo con Sandbox
 
-Si quieres profundizar y comenzar a jugar con single-file components, visita [esta aplicación to-do de ejemplo ](https://codesandbox.io/s/o29j95wx9) en CodeSandbox.
+Si quieres profundizar y comenzar a jugar con Componentes de un Solo Archivo, visita [esta aplicación to-do de ejemplo](https://codesandbox.io/s/o29j95wx9) en CodeSandbox.
 
-### Para usuarios nuevos en Sistemas de Construcción de Módulos en JavaScript
+### Para usuarios nuevos en Sistemas de Compilación de Módulos en JavaScript
 
-Con componentes `.vue`, estamos entrando en el reino de aplicaciones JavaScript avanzadas. Eso significa aprender a usar algunas herramientas adicionales si aún no lo ha hecho:
+Con los componentes `.vue` entramos en el reino de las aplicaciones JavaScript avanzadas. Esto significa aprender a usar algunas herramientas adicionales, si es que aún no lo has hecho:
```

Comment:
Following the same direction, we should use `ha hecho` here..

Reply:
Changed to formal treatment throughout the document.